ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__multi-nightly.yaml

```diff
@@ -493,6 +493,52 @@ tests:
     test:
     - ref: openshift-extended-test
     workflow: cucushift-installer-rehearse-aws-ipi-sno-lvms
+- as: aws-ipi-sts-mini-perm-amd-f28-destructive
```

Comment:
I am thinking if we can replace some existing sts ci jobs with sts + mini-perm to run the coverage to save the newly added jobs.

Reply:
`aws-ipi-proxy-sts-amd (arm)`  is one job can be updated, but for private-s3 workflow, I think we could add a new job, as we don't have an IPI cluster with installer provisioned VPC yet, with this new workflow, we can tests ccoctl + ipi cluster both with minimum permission.

Reply:
I guess `aws-ipi-disc-priv-sts-ep` can be updated to enable mini-perm to cover private-s3 workflow.

Reply:
There are too many disc-priv STS cluster profiles, if possible, I'd like to merge `aws-ipi-disc-priv-sts-ep` and `aws-ipi-disc-priv-sts-efs`, and add one new job `aws-ipi-sts-priv-s3-mini-perm`

Reply:
>  I'd like to merge aws-ipi-disc-priv-sts-ep and aws-ipi-disc-priv-sts-efs

I am okay with that. 

> add one new job aws-ipi-sts-priv-s3-mini-perm

But for mini-perm job, I have a bit different opinion, actually per my understanding, we are supposed to run most of ci jobs using mini-perm, after all, using mini-perm account is easier to find out issues than using a full-permission account. Of course, if you prefer to add the new job to cover "an IPI cluster with installer provisioned VPC yet" beside mini-perm, I am okay with that, too.